Website/src/items/renderings/edgewebsite/Sessions/SessionsGrid.yml

```diff
@@ -54,6 +57,9 @@ SharedFields:
                       name: field(name: "name") {
                         value
                       }
+                      url {
+                        path
```

Comment:
/sessions/Intro-to-Cycling---Sponsored-by-Striva

This is why this was not being used. I was thinking of removing dashes from item name, which could solve issue.